doc/howto/json.md

```diff
@@ -45,9 +45,9 @@ for path in paths:
 %pip install jupysql duckdb duckdb-engine rich --quiet
 ```
 
-Now, let's generate some data. Note that DuckDB expects your data to contain *one JSON object per line*; this format is called [JSON Lines](https://jsonlines.org/), and it often comes with the `.json`, `.jsonl.gz`, or `.jsonl.bz2` extension.
+Now, let's generate some data.
 
-Our sample data contains four rows:
+We'll write it in typical JSON format as well as [JSON Lines](https://jsonlines.org/). JSON Lines, or newline-delimited JSON, is a structured file format in which each individual line is a valid JSON object, separated by a newline character (`/n`). Our sample data contains four rows:
 
 ```{code-cell} ipython3
 from pathlib import Path
```

Comment:
2 notes
1. I'm not sure data is a standard json file and it might be a little confusing
```
data = [
{}, {}, {}
]
vs
data = {
{}, {}, {}
}
vs
{
  data : [
  {}, {}, {}
  ]
}
```
2. Do you think we should split json declaration and the formatting logic into 2 different code cells to make it easier to read, copy the code, and see the differences between the jsons? The 1st part will declare the json and print it, the 2nd part will print the formatted json.

Reply:
For 2, so you mean print `data` after declaring it in one cell and then start a new cell with declaring the `lines` variable?

Reply:
Yes, I think it will be easier to understand that we have a simple json file (step 1) and then we are formatting it (step 2). A different cell for every step.

Reply:
I think when rewriting this tutorial we wanted to keep this format as an array of objects because DuckDB just recently released an update that allowed for that. I just made the changes to fix the second point, let me know